tweakwcs/correctors.py

```diff
@@ -474,9 +474,9 @@ def tanp_to_det(self, x, y):
             ra = np.atleast_1d(ra)
             dec = np.atleast_1d(dec)
         x, y = self._wcs.all_world2pix(ra, dec, 0, tolerance=1e-6, maxiter=50)
-        if not ndim:
-            x = float(x)
-            y = float(y)
+        if not ndim and x.size == 1:
```

Comment:
@mcara what are your thoughts on sneaking this into #185 vs a separate PR? If a separate PR is preferable I'm happy to open one.

Reply:
I think it is fine to do it here. Just make a suggestion.

Reply:
Sigh... I meant in #185

Reply:
I don't think there is a way to make a suggestion on an unchanged file :-/ perhaps there is a way I'm not aware of?

Reply:
yeah, you are right. I'll add this